### DISCUSSION TITLE
Duplicate items in the new feed

### DISCUSSION THREAD

**Discussion by scimas** [community/community#13391] [Other Feature Feedback, Questions, & Ideas]
The new feed gets a lot of duplicate content. Most of the cards in my feed right now are like this:
```
User A starred repository R
```
```
User A added repository R to list L
```
where most of the user As are people I don't follow or interact with.

Please don't make this feed the only one available or even the default one. But if it is here to stay, please make it less noisy.

**Comment by Drjacky** [comment]
Something similar happened to me:
> UsernameAAA starred https://github.com/Drjacky/MVVMTemplate and https://github.com/Drjacky/MVVMTemplate
